pkg/controllers/policyrc/controller.go

```diff
@@ -279,6 +257,7 @@ func (c *Controller) reconcileCount(qualifiedName common.QualifiedName) (status
 }
 
 func (c *Controller) reconcilePersist(
+	ctx context.Context,
 	metricName string,
 	qualifiedName common.QualifiedName,
 	nsScopeStore, clusterScopeStore cache.Store,
```

Comment:
At line 287 below, logger should be obtained from the context. `logger := klog.FromContext(ctx)` and it should already contain the object tag.